capstone/src/main/java/com/google/sps/data/RestaurantQueryHelper.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps.data;
+
+import com.google.maps.model.PlaceDetails;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Arrays;
+import java.util.Collections;
+import com.google.sps.data.Restaurant;
+
+
+public final class RestaurantQueryHelper {
+  public static int MAX_RESULTS = 20;
+
+  public boolean restaurantContainsKeyword( Entity RestaurantEntity, String keywordsCombinedString) {
+    Set<String> keywords = splitStringToSet(keywordsCombinedString);
+
+    List<String> restauarantTagsAsList = (List<String>) RestaurantEntity.getProperty("tags");
+    Set<String> currRestaurantTags = new HashSet<String>(); 
+    currRestaurantTags.addAll(restauarantTagsAsList); 
+
+    if (!Collections.disjoint(currRestaurantTags, keywords)) {
+      return true;
+    }
+    else return false;
```

Comment:
could this just be return false

Reply:
good catch! changed to just `return (!Collections.disjoint(currRestaurantTags, keywords))` in PR(#66)